modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/AddWriteCommittedInvokeClosure.java

```diff
@@ -81,6 +81,14 @@ public void call(@Nullable VersionChain oldRow) throws IgniteInternalCheckedExce
             throw new StorageException("Write intent exists: [rowId={}, {}]", oldRow.rowId(), storage.createStorageInfo());
         }
 
+        if (row == null && oldRow == null) {
+            // If previous version doesn't exist and current version is a tombstone,
```

Comment:
I would paraphrase in:
`If only one version, and it is a tombstone, then we will not save such a chain.`